osgp/platform/osgp-adapter-ws-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/ws/da/application/config/PersistenceConfigCore.java

```diff
@@ -20,7 +21,7 @@
 import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
 import org.springframework.transaction.annotation.EnableTransactionManagement;
 
-import com.zaxxer.hikari.HikariDataSource;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this duplicated import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

Reply:
Done